Onepunch/README.md

```diff
@@ -1,26 +1,50 @@
 # Onepunch
+
 ## Screenshots
-![Home](./__Home.png)
-![Profile](./__Profile.png)
-![Playlist](./__Playlist.png)
+
+![dark_home](./screenshots/display.gif)
 
 ## More
 
-#### About
-A simple gruvified spotify theme. 
+### About
+
+A simple gruvified spotify theme. Move to [Extra Preview](#extra-preview) for more screenshots.
+
+### Information
+
+- To apply this theme:
+
+  ```shell
+  # for dark theme
+  spicetify config current_theme Onepunch color_scheme dark
+  spicetify apply
+
+  #for light theme
+  spicetify config current_theme Onepunch color_scheme light
+  spicetify apply
+
+  ```
+
+- Tested on macOS only and pretty much everything worked. ﾍ(･\_|
+
+### Extra Preview
```

Comment:
I would prefer all READMEs to be consistent, so I would suggest to rename this section to `Screenshots`

Reply:
done